modules/@apostrophecms/module/index.js

```diff
@@ -782,13 +782,15 @@ module.exports = {
       determineBestAssetUrl(name) {
         let urlOption = self.options[`${name}Url`];
         const imageOption = self.options[`${name}Image`];
-        if (imageOption) {
-          const chain = [ ...self.__meta.chain ].reverse();
-          for (const entry of chain) {
-            const path = `${entry.dirname}/public/${name}.${imageOption}`;
-            if (fs.existsSync(path)) {
-              urlOption = `/modules/${entry.name}/${name}.${imageOption}`;
-              break;
+        if (!urlOption) {
```

Comment:
The existing placeholderUrl option for images should still take precedence if explicitly present, otherwise placeholderImage is considered. Caught by unit tests